project/Dependencies.scala

```diff
@@ -99,7 +109,7 @@ object Dependencies {
       pagopa.commons                     % Compile,
       pagopa.jwt                         % Compile,
       pagopa.signer                      % Compile
-    )
+    ).map(_.withSources.withJavadoc)
```

Comment:
Never use? Which is its purpose?

Reply:
It forces the download of javadoc and sources when you compile the project. They're ofc not boxed in the final jar, they're simply handy to use in IDEs.